pkg/controller/signup.go

```diff
@@ -52,28 +47,10 @@ func (s *Signup) PostHandler(ctx *gin.Context) {
 	ctx.Writer.WriteHeaderNow()
 }
 
-// UpdateVerificationHandler starts the verification process and updates a usersignup resource. The
+// InitVerificationHandler starts the verification process and updates a usersignup resource. The
 // ctx should contain a JSON body in the request with a country_code and a phone_number string
```

Comment:
Can we add a bit more details about what this function does.

Reply:
I've added more detail here, and also on the `InitVerification()` function in the Verification service.